### ISSUE TITLE
TC358743 broken on 5.4 kernel

### ISSUE THREAD

**Issue by mdevaev** [pikvm/pikvm#17]
I use a RPi 4B. When I try to reduce the stream quality, it doesn't work. From dmesg:

````
[  178.270874] 1f40: 00000000 00088cb8 eff550f4 c021a420 00000000 c1400000 c1404fb0 c1404ff8
[  178.288902] 1f60: c14d9580 c126ce48 30c5387d 00000000 c1500e10 c1401f90 c0209268 c020926c
[  178.306991] 1f80: 60070013 ffffffff
[  178.320471] [<c0201a78>] (__irq_svc) from [<c020926c>] (arch_cpu_idle+0x38/0x3c)
[  178.337882] [<c020926c>] (arch_cpu_idle) from [<c025ef38>] (do_idle+0xc4/0x148)
[  178.355092] [<c025ef38>] (do_idle) from [<c025f2a8>] (cpu_startup_entry+0x18/0x20)
[  178.372530] [<c025f2a8>] (cpu_startup_entry) from [<c1200f90>] (start_kernel+0x500/0x540)
[  178.480634] i2c-bcm2835 fe205000.i2c: i2c transfer timed out
[  178.495735] tc358743 10-000f: i2c_wr: writing register 0x500 from 0xf failed
[  179.520647] i2c-bcm2835 fe205000.i2c: i2c transfer timed out
[  179.535793] tc358743 10-000f: i2c_wr: writing register 0x500 from 0xf failed
[  180.560663] i2c-bcm2835 fe205000.i2c: i2c transfer timed out
[  180.575765] tc358743 10-000f: i2c_wr: writing register 0x500 from 0xf failed
[  181.600674] i2c-bcm2835 fe205000.i2c: i2c transfer timed out
[  181.615807] tc358743 10-000f: i2c_wr: writing register 0x500 from 0xf failed
[  181.633387] tc358743 10-000f: i2c_rd: reading register 0x14 from 0xf failed
````

Btw, I have a B101 Revision 4. I think that the RPi or the B101 could have a problem. What do you think?

_Originally posted by @stsirakidis in https://github.com/pikvm/pikvm/issues/16#issuecomment-624284789_

**Comment by mdevaev** [comment]
TLDR
The new 5.x kernel is broken at several points at once (i2c, initrd build, etc.), so for now I have forced the use the 4.x. After some time, the 5.x core will be fixed and I will remove this restriction. RPi forums say that the new release of the kernel is experimental. So let's live on the old kernel for now.

**Comment by mdevaev** [comment]
Related: https://github.com/raspberrypi/linux/issues/3602

**Comment by mdevaev** [comment]
Fixed at the latest kernel. Now we have another 5.4 problem: https://github.com/pikvm/pikvm/issues/20
